jx/bdd/boot-vault/cluster.yaml

```diff
@@ -15,4 +15,3 @@ clusters:
       - command: jx
         args:
           - boot
-          - -b
```

Comment:
Why did you remove this? I think this allows to run boot in batch mode, otherwise it might ask for interactive input. I am not sure if it makes any difference.

Reply:
ahh, my wrong. You are right. Let me remove this.

Reply:
I was thinking only of -b as a option to boot which does not exist. But in this case it is a global option `jx -b boot`.

Reply:
Fixed. Good catch. That said, I don't think it makes a difference in our case.